iphone/Classes/TiAppiOSUserActivityProxy.m

```diff
@@ -425,6 +438,73 @@ - (void)resignCurrent:(id)unused
   }
   [_userActivity resignCurrent];
 }
+
+#if IS_XCODE_10
+
+- (NSNumber *)eligibleForPrediction
+{
+  if (![TiUtils isIOSVersionLower:@"12.0"]) {
+    return NUMBOOL(NO);
+  }
+
+  return NUMBOOL(_userActivity.isEligibleForPrediction);
```

Comment:
We should move all these to native Obj-C 2.0 statements, e.g. `@(_userActivity.isEligibleForPrediction)` and `@(NO)`. But that should be done with an own PR and an own ticket. Maybe we can deprecate the old macros in SDK 8.0.0?.